src/cli.rs

```diff
@@ -772,35 +820,38 @@ pub fn handle_cli_edge_app_command(command: &EdgeAppCommands) {
         EdgeAppCommands::List { json } => {
             handle_command_execution_result(edge_app_command.list(), json);
         }
-        EdgeAppCommands::Upload { path } => {
-            match edge_app_command.upload(transform_edge_app_path_to_manifest(path).as_path()) {
-                Ok(()) => {
-                    println!("Edge app successfully uploaded.");
+        EdgeAppCommands::Upload { path, app_id } => {
+            match edge_app_command.upload(
+                transform_edge_app_path_to_manifest(path).as_path(),
+                app_id.clone(),
```

Comment:
Shouldn't here also be get_actual_app_id?

Reply:
And for this function I still fail if manifest does not exist